db/src/models/rating.rs

```diff
@@ -102,6 +102,27 @@ impl Rating {
             .await?)
     }
 
+    pub async fn history_for_uuid(
+        uuid: &Uuid,
+        game_speed: &GameSpeed,
+        conn: &mut DbConn<'_>,
+    ) -> Result<Vec<Self>, DbError> {
+        let game_speed = match game_speed {
+            GameSpeed::Untimed => GameSpeed::Correspondence.to_string(),
```

Comment:
Untimed games are always unranked/unrated so they don't make sense to be grouped into Correspondence in my mind.